src/interfaces/stylesheet/__init__.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from ..util import load_path
+
+def read_stylesheet(file_name):
+    path = load_path(os.getcwd() + "/interfaces/stylesheet/", file_name)
+    return open(path).read() if path else None
+
+
+def read_icon(file_name):
+    return load_path(os.getcwd() + "/img/", file_name)
```

Comment:
I'm not positive, but I think will likely won't work with once the application has been compiled into a binary. You might want to use a filepath `join` or `abspath`. We'll also need to update the build module to inject these dependencies when building but that's definitely outside of the scope of this PR.

Reply:
Agree. we can come back to this when we do some codebase optimization.